docs/jobs/profiles_downloader.md

```diff
@@ -47,6 +47,17 @@ See this guide on [how to generate the qdt-files.json](../usage/profile.md#gener
     source: file:///home/jmo/Git/Geotribu/profils-qgis
 ```
 
+With repository on [UNC path](https://en.wikipedia.org/wiki/Path_(computing)#Uniform_Naming_Convention):
+
+```yaml
+- name: Download profiles from Git repository stored on a local and "hidden" network drive
+  uses: qprofiles-downloader
+  with:
+    branch: main
+    protocol: git_local
+    source: file://\\srv-01\QDT$
```

Comment:
Side note: I think this convention used in QDT is not standard (https://en.wikipedia.org/wiki/File_URI_scheme)

Reply:
From your article:

> There are two ways that Windows UNC filenames (such as \\server\folder\data.xml) can be represented.

Well, we give here a sample in YAML where it's casted as pure string converted later in path formalism.